stdlib/tests/crypto/falcon.rs

```diff
@@ -83,21 +85,49 @@ fn test_falcon512_diff_mod_q() {
         exec.rpo_falcon512::diff_mod_q
     end
     ";
-
-    let u = rand::thread_rng().gen_range(0..J);
-    let v = rand::thread_rng().gen_range(Q..M);
+    let v = rand::thread_rng().gen_range(0..Felt::MODULUS);
+    let (v_lo, v_hi) = (v as u32, v >> 32);
     let w = rand::thread_rng().gen_range(0..J);
+    let u = rand::thread_rng().gen_range(0..J);
```

Comment:
We should move away from using randomly-generated values in unit tests; if we randomly fall on a failing value, we won't be able to reproduce. Broadly speaking, unit tests should test specific hardcoded values, while prop tests should do the "sweeping" (as you're doing below)

Reply:
Good point, with proptest, the above is redundant as is. Changed it now to be deterministic